bundle/src/main/java/com/adobe/acs/commons/util/impl/WorkflowHelperImpl.java

```diff
@@ -48,7 +47,6 @@
 import java.util.Map;
 
 @Component
-@Service
```

Comment:
Unfortunately it's not obvious if this change comes with the PR or is caused by the merge/rebasing :-(

Reply:
I changed this file to use OSGi annotations (instead of Felix) therefor the service annotations are not required anymore (implicit via the interface). I can add it to make this more clear though.

I changed this file because locally I was having problems w Felix scr annotated services no generating OSGi-INF metadata XML files. Im not sure if other's are experiencing the same?